stock_location_orderpoint/models/stock_location_orderpoint.py

```diff
@@ -0,0 +1,336 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from copy import copy
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.osv import expression
+from odoo.tools import float_compare, groupby, split_every
+
+
+class StockLocationOrderpoint(models.Model):
+    _name = "stock.location.orderpoint"
+    _description = "Stock location orderpoint"
+    _order = "sequence"
+
+    name = fields.Char(
+        "Name",
+        copy=False,
+        required=True,
+        readonly=True,
+        default=lambda self: self.env["ir.sequence"].next_by_code(
+            "stock.location.orderpoint"
+        ),
+    )
+    company_id = fields.Many2one(
+        "res.company",
+        "Company",
+        required=True,
+        default=lambda self: self.env.company,
+    )
+    location_id = fields.Many2one(
+        "stock.location",
+        "Location",
+        ondelete="cascade",
+        required=True,
+        check_company=True,
+        domain='[("company_id", "=", company_id)]',
+    )
+    trigger = fields.Selection(
+        [("auto", "Auto"), ("manual", "Manual")],
+        "Trigger",
+        default="auto",
+        required=True,
+    )
+    replenish_method = fields.Selection(
+        [("fill_up", "Fill up")],
+        default="fill_up",
+        required=True,
+        help="Defines how the qty to replenish is computed",
+    )
```

Comment:
@jbaudoux or @rousseldenis do you have maybe a better name for this field and value.
It is used in _get_qty_to_replenish to get the right method to compute the qty to replenish

Reply:
That seems good to me. What is bothering you ?

Reply:
Is not bothering me, in German i would find an exact description for it, but in english if find sometimes hard.